src/app/devices/deviceContent/sagas/digitalTwinInterfacePropertySaga.ts

```diff
@@ -67,7 +67,8 @@ export function* patchDigitalTwinInterfacePropertiesSaga(action: Action<PatchDig
     }
 }
 
-export const generateInterfacePropertiesPayload = (interfaceName: string, propertyKey: string, patchData: object) => {
+// tslint:disable-next-line:no-any
+export const generateInterfacePropertiesPayload = (interfaceName: string, propertyKey: string, patchData: any) => {
```

Comment:
Is this type changed only for the test? Could we supply an object for testing instead of a number?

Reply:
It is more like a bug discovered by the test. The value can be anything not just an object :)